lib/utils.pm

```diff
@@ -2826,4 +2827,21 @@ sub define_secret_variable {
     script_run("set +a");
 }
 
+=head2 ping_size_check
+    ping_size_check($target);
+ping_size_check will ping defined target with different and increasing size with disabled
+packet fragmentation.
+=cut
+
+sub ping_size_check {
+    my $target = shift;
+    # Check connectivity with different packet size to target
+    # Fragmentation is disabled, maximum size is 1430 to fit in 1458 MTU in GRE tunel
+    my @size = (100, 1000, 1350, 1400, 1430);
+    foreach (@size) {
+        assert_script_run("ping -M do -s $_ -c 1 $target");
```

Comment:
I suggest to add a "fail_message" here, see http://open.qa/api/testapi/#_assert_script_run, so that the consequence of a failure can be more clear to reviewers, e.g.

```suggestion
        assert_script_run("ping -M do -s $_ -c 1 $target", fail_message => "ping with packet size $_ failed, problems with MTU size in multi-machine GRE tunnel setup expected");
```